pkg/subctl/cmd/validate_firewall.go

```diff
@@ -0,0 +1,30 @@
+/*
+© 2021 Red Hat, Inc. and others.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var validateFirewallConfigCmd = &cobra.Command{
+	Use:   "firewall",
+	Short: "Validate the Firewall Configuration in the cluster.",
```

Comment:
nit: I don't think `Firewall Configuration`  should be capitalized but at the very least  we should be consistent wrt case - in other places these aren't capitalized.

Reply:
Oops, I missed it. Thanks @tpantelis